apps/communications/contacts/js/bootstrap.js

```diff
@@ -33,6 +33,7 @@
        '/shared/js/contacts/import/utilities/config.js',
        '/contacts/js/utilities/extract_params.js',
        '/contacts/js/utilities/cookie.js',
+       '/contacts/js/utilities/object.js',
```

Comment:
Agree. I didn't want to make more than minimal changes for this, but I also think it's better to load it when needed.